### PR TITLE
DOC-11702: IA Phase 2 — Backport IA refresh to server

### PR DESCRIPTION
**Pull request by simon-dew**
Docs issue: DOC-11702

Preview: [Developer Intro](https://preview.docs-test.couchbase.com/DOC-11702/server/current/develop/intro.html)
Credentials: [Preview docs for internal review](https://couchbasecloud.atlassian.net/wiki/x/dYF_dQ#Preview-docs-for-internal-review)

This PR is for docs-server. It updates the [docs-server/DOC-11965-ia-refresh-76](https://github.com/couchbase/docs-server/tree/DOC-11965-ia-refresh-76) feature branch.

That feature branch must be merged at the same time as the [docs-devex/DOC-11965-ia-refresh-76](https://github.com/couchbaselabs/docs-devex/tree/DOC-11965-ia-refresh-76) feature branch.
The following related PR updates that feature branch:

* https://github.com/couchbaselabs/docs-devex/pull/175

These feature branches must be merged at the same time as the following PRs:

* https://github.com/couchbase/docs-analytics/pull/20
* https://github.com/couchbaselabs/docs-devex/pull/176
* https://github.com/couchbasecloud/couchbase-cloud/pull/30186

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -135,7 +126,7 @@ include::vector-search:partial$nav.adoc[]
   *** xref:fts:fts-advanced-settings-CBFT-ENV-OPTIONS.adoc[CBFT_ENV_OPTIONS]
  ** xref:fts:fts-monitor.adoc[Statistics and Monitoring]
  ** xref:fts:fts-troubleshooting.adoc[Troubleshooting and FAQs]
-* xref:eventing:eventing-overview.adoc[Eventing]
+* xref:eventing:eventing-overview.adoc[Run a Function on Data Change]
```
Comment (bfavini) [inline modules/ROOT/nav.adoc:23]: @simon-dew  In Capella, this nav entry is "Add Eventing to your application" (Which I happen to like better). Any reason it's different here?

Comment (simon-dew) [reply modules/ROOT/nav.adoc:23]: This was the title from Dina's IA plan, as discussed with Jon Strabala, which I don't think was ever implemented.

File: modules/tools/pages/.tools-ref.adoc
Comment (ggray-cb) [inline modules/tools/pages/.tools-ref.adoc:1]: Is there a reason to hide rather than just deleting?

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (bfavini) [reply modules/ROOT/nav.adoc:23]: would like to revisit this at some point..but not today...

**Review by bfavini — APPROVED** [review]

File: modules/tools/pages/.tools-ref.adoc
Comment (simon-dew) [reply modules/tools/pages/.tools-ref.adoc:1]: Good point!

**Pull request merged**